graph/pull_v2.go

```diff
@@ -76,6 +76,9 @@ func (p *v2Puller) pullV2Repository(tag string) (err error) {
 	broadcaster.Add(p.config.OutStream)
 	if found {
 		// Another pull of the same repository is already taking place; just wait for it to finish
+		if dryRun {
+			fmt.Printf("!!! Another pull of the same image is already running, dry-run is not possible unless you restart the Docker daemon !!!\n")
```

Comment:
This output needs to be directed to `p.config.OutStream`, using `p.sf.FormatStatus`, instead of printing to the daemon's stdout.